test/augmentation/test_augmentation.py

```diff
@@ -3398,6 +3398,24 @@ def test_module(self, device, dtype):
         img = torch.ones(1, 3, 5, 5, device=device, dtype=dtype)
         self.assert_close(op(img, *func_params), op_module(img))
 
+    def test_module_kernel_float(self, device, dtype):
+        func_params = [3, torch.tensor([1.5, 1.5]).view(1, -1)]
+        params = [3, (1.5, 1.5)]
```

Comment:
the idea isn't the kernel be a float here?

Reply:
My mistake, the tests are now what they are supposed to be: int or tuple of int for the kernel, and sigma as a tuple of float or a tensor of float.